osu.Game/Overlays/Settings/SettingsNumberBox.cs

```diff
@@ -2,16 +2,20 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Graphics;
-using osu.Game.Graphics.UserInterface;
 
 namespace osu.Game.Overlays.Settings
 {
     public class SettingsNumberBox : SettingsItem<string>
     {
-        protected override Drawable CreateControl() => new OsuNumberBox
+        protected override Drawable CreateControl() => new OsuSettingsNumberBox
         {
             Margin = new MarginPadding { Top = 5 },
             RelativeSizeAxes = Axes.X,
         };
+
+        public class OsuSettingsNumberBox : SettingsTextBox.OsuSettingsTextBox
```

Comment:
I think just stating `NumberBox` is enough, same goes for `TextBox`. (since you're always gonna do `SettingsTextBox.TextBox` and `SettingsNumberBox.NumberBox`